src/passes/MemoryPacking.cpp

```diff
@@ -28,12 +28,19 @@ struct MemoryPacking : public Pass {
   bool modifiesBinaryenIR() override { return false; }
 
   void run(PassRunner* runner, Module* module) override {
-    // Conservatively refuse to change segments if bulk memory is enabled to
-    // avoid invalidating segment indices or segment contents referenced from
+    if (!module->memory.exists)
+      return;
```

Comment:
`{ }` if not on the same line.

@aheejin didn't we have clang-tidy check for that?

Reply:
We do. Travis CI is pending now, but once it runs it will error out.

Reply:
Thanks, and sorry I didn't notice it was still pending.